tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -326,6 +326,34 @@ public void AcquireTokenSameUserTest()
             }
         }
 
+        [TestMethod]
+        public void AcquireTokenWithDefaultRedirectURITest()
+        {
+            using (var harness = CreateTestHarness())
+            {
+                //harness.HttpManager.AddInstanceDiscoveryMockHandler();
+                PublicClientApplication app = PublicClientApplicationBuilder.Create(MsalTestConstants.ClientId)
+                                                                            .WithAuthority(new Uri(ClientApplicationBase.DefaultAuthority), true)
+                                                                            .BuildConcrete();
+                //Validate legacy default uri
+                Assert.AreEqual(app.AppConfig.RedirectUri, "urn:ietf:wg:oauth:2.0:oob");
+
+                app = PublicClientApplicationBuilder.Create(MsalTestConstants.ClientId)
+                                                                            .WithAuthority(new Uri(ClientApplicationBase.DefaultAuthority), true)
+                                                                            .WithHttpManager(harness.HttpManager)
+                                                                            .WithTelemetry(new TraceTelemetryConfig())
+                                                                            .WithDefaultRedirectUri()
+                                                                            .BuildConcrete();
+
+                //Validate new default redirect uri
+#if DESKTOP
+                Assert.AreEqual(app.AppConfig.RedirectUri, "https://login.microsoftonline.com/common/oauth2/nativeclient");
+#elif NET_CORE
+                Assert.AreEqual(app.AppConfig.RedirectUri, "https://localhost");

```

Comment:
>Assert.AreEqual(app.AppConfig.RedirectUri, "https://localhost"); [](start = 16, length = 64)

I'm not convinced this is right.

Reply:
@henrik-me would you mean that we should not propose localhost for .NET Core? or do you also question UWP?

Reply:
At this time I'm talking about netcore. UWP I believe has a different schema?

Reply:
Will be address in a later PR